docs/README.md

```diff
@@ -110,10 +111,46 @@ functions:
     env:
       MY_VARIABLE: "my-value"
 ```
+### Managing containers
+
+**Requirements:** You need to have Docker installed to be able to push your image to your Scaleway registry.
+
+Your containers must be defined in a `containers` field in the `custom` field of your manifest. You need to specify the relative path of your container directory in `directory` containing the Dockerfile and any additional files :
```

Comment:
You must define your containers inside the `custom.containers` field in your `serverless.yml` manifest.
Each container must specify the relative path its application directory (containing the Dockerfile, and all files related to the application to deploy):